release/docker/Dockerfile

```diff
@@ -1,36 +1,16 @@
-FROM alpine:3.12
-
-ENV LANG=en_US.UTF-8
+FROM python:3.9-slim-buster
 
 ARG WHEEL_MITMPROXY
 ARG WHEEL_BASENAME_MITMPROXY
 
+RUN useradd -mU mitmproxy
+RUN apt-get update \
+    && apt-get install -y gosu \
+    && rm -rf /var/lib/apt/lists/*
+    
 COPY $WHEEL_MITMPROXY /home/mitmproxy/
-
-# Add our user first to make sure the ID get assigned consistently,
-# regardless of whatever dependencies get added.
-RUN addgroup -S mitmproxy && adduser -S -G mitmproxy mitmproxy \
-    && apk add --no-cache \
-        su-exec \
-        git \
-        g++ \
-        libffi \
-        libffi-dev \
-        libstdc++ \
-        openssl \
-        openssl-dev \
-        python3 \
-        python3-dev \
-    && python3 -m ensurepip --upgrade \
-    && pip3 install -U pip \
-    && LDFLAGS=-L/lib pip3 install -U /home/mitmproxy/${WHEEL_BASENAME_MITMPROXY} \
-    && apk del --purge \
-        git \
-        g++ \
-        libffi-dev \
-        openssl-dev \
-        python3-dev \
-    && rm -rf ~/.cache/pip /home/mitmproxy/${WHEEL_BASENAME_MITMPROXY}
+RUN pip3 install --no-cache-dir -U /home/mitmproxy/${WHEEL_BASENAME_MITMPROXY} \
+    && rm -rf /home/mitmproxy/${WHEEL_BASENAME_MITMPROXY}
```

Comment:
@mhils maybe `rm -rf /home/mitmproxy/${WHEEL_BASENAME_MITMPROXY}` can also be removed as it won't save any disk space, because it's not added in this image layer, unless we dynamically download add add it, what do you think?

Reply:
I think we really want to use the local wheel, so downloading is bad. I'm indifferent regarding this line. Space-wise it doesn't matter, we either save a) one line in the Dockerfile or b) one file in the container filesystem. Either option is good with me, whatever you prefer. I think you are way more familiar with Docker conventions. :)